api/types/management.go

```diff
@@ -34,15 +34,16 @@ type AccessTokensDeleteResponse struct {
 }
 
 type Account struct {
-	UserID         string    `json:"userID"`
-	Email          string    `json:"email"`
-	GithubID       int32     `json:"githubID"`
-	GithubUsername string    `json:"githubUsername"`
-	DateJoined     time.Time `json:"dateJoined"`
-	Credit         string    `json:"credit"`
-	FirstName      string    `json:"firstName"`
-	LastName       string    `json:"lastName"`
-	Providers      []string  `json:"providers"`
+	UserID              string    `json:"userID"`
+	Email               string    `json:"email"`
+	GithubID            int32     `json:"githubID"`
+	GithubUsername      string    `json:"githubUsername"`
+	DateJoined          time.Time `json:"dateJoined"`
+	Credit              string    `json:"credit"`
+	FirstName           string    `json:"firstName"`
+	LastName            string    `json:"lastName"`
+	Providers           []string  `json:"providers"`
+	GithubCredentialsID *string   `json:"-,omitempty"`
```

Comment:
What is the `GithubCredentialsID` here?

Reply:
this is the address we store the credentials ID for the user, it's part of a user's `account`. no i don't think we need to expose it to the user but then we have diverging models for a user account (in infra) and the user account (in `unweave/unweave`) ; any thoughts on how we can resolve this?

Reply:
I think we can just use the db types to get the `GithubCredentialsID` in infra.